types/node/ts3.1/crypto.d.ts

```diff
@@ -545,6 +551,16 @@ declare module "crypto" {
         };
     }
 
+    interface X25519KeyPairOptions<PubF extends KeyFormat, PrivF extends KeyFormat> {
+        publicKeyEncoding: {
+            type: 'pkcs1' | 'spki';
+            format: PubF;
+        };
+        privateKeyEncoding: BasePrivateKeyEncodingOptions<PrivF> & {
+            type: 'sec1' | 'pkcs8';
```

Comment:
docs tell that `sec1` is only for EC. Is it also allowed for X25519?

Reply:
Same as above. Sorry again.